pipelines/datasets/br_me_rais/constants.py

```
@@ -0,0 +1,282 @@
+
+    ESTAB_FILE = "RAIS_ESTAB_PUB.7z"
+
+    VINCULOS_FILES = [
+        "RAIS_VINC_PUB_CENTRO_OESTE.7z",
+        "RAIS_VINC_PUB_MG_ES_RJ.7z",
+        "RAIS_VINC_PUB_NI.7z",
+        "RAIS_VINC_PUB_NORDESTE.7z",
+        "RAIS_VINC_PUB_NORTE.7z",
+        "RAIS_VINC_PUB_SP.7z",
+        "RAIS_VINC_PUB_SUL.7z",
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)